artifactory/commands/utils/templateutils.go

```diff
@@ -38,6 +38,39 @@ func ConvertTemplateToMap(tuc TemplateUserCommand) (map[string]interface{}, erro
 	return configMap, errorutils.CheckError(err)
 }
 
+func ConvertTemplateToMaps(tuc TemplateUserCommand) ([]map[string]interface{}, error) {
```

Comment:
nit pick: It will be hard to guess tuc is better to expand it to templateUserCommand